### ISSUE TITLE
Incorrect field name in .utc and .bic GFF structures

### ISSUE THREAD

**Issue by LaputianBird** [Beamdog/nwn-issues#16]
**To Reproduce**

Create a new creature in the toolset, or a new character in game. Open the .utc or .bic with a GFF editor

**Specifics**

 - OS: Windows (likely any OS)
 - Game build: Any
 - Distro platform: Any

**describe the bug**

The engine writes one of the appearance parts with an incorrect field name (it follows the naming convention for armor parts, not that for body parts).
![sshot_2](https://user-images.githubusercontent.com/60814252/77940182-981fef00-72b8-11ea-8b89-14e2bad9f290.png)

**Comment by niv** [comment]
What do you envision the fix here looks like?

Renaming the data field will bring nothing but grief. This is not pretty, but just a visual/textual oddity, so leaving it well be seems best to me.

**Comment by LaputianBird** [comment]
I stumbled on it while manipulating blueprints from a script, and it definitely looks wrong, hence why I reported it. Just in case.
I'm not against a *won't fix* sentence, I just thought there was the option to handle it as an alias of `BodyPart_RFoot` on load, and set the proper field name on save, so that new files are fixed and older files are still valid.
Either way, easy to handle it as an exception in tools/scripts.
